recipes/kibana/.dockerignore

```diff
@@ -0,0 +1 @@
+Dockerfile
```

Comment:
The docker build context is very important when it comes to building images. It is the sub-set of files that you can use in your Dockerfile with the **ADD** or **CP** commands.

The docker image is not build on your machine but on the Docker registry you are talking to. When you run the `docker build` command, then the entire context is copied to the Docker registry, then the registry starts building your image.

So you definitely want to avoid having unnecessary files to the Docker build context, and that is what the `.dockerignore` file is meant to.

I did 4e09d06 before knowing that Quay.io expects the `Dockerfile` to be in the build context. So yes, you can remove the `Dockerfile` from the `.dockerignore`